### ISSUE TITLE
Add learning section on the community page

### ISSUE THREAD

**Issue by starbelly** [erlef/website#237]
A learning section should be added to the community page consisting of sub-sections such as (but not limited to) : 

   - courses
   - tutorials
   - books
   
Let's try to gather resources in this issue first 🚀

**Comment by juhalehtonen** [comment]
Should we make a distinction between paid and free materials on the website?

Either way, to kick off the listing, here's a few resources I've run into:

* [Learn You Some Erlang for Great Good](https://learnyousomeerlang.com/) is a classic Erlang text
* [Stuff goes bad: Erlang in Anger](https://www.erlang-in-anger.com/) is another classic text, but has a very specific focus on production systems
* [Learning section of elixir-lang.org](https://elixir-lang.org/learning.html) does a great job compiling a collection of learning materials related to the language (books, podcasts, courses and more)
* [Pragmatic Programmers](https://pragprog.com/categories/elixir-phoenix-and-otp/) releases numerous books on Erlang/Elixir
* [LFE website](https://lfe.io/) lists a selection of books covering the language

**Comment by starbelly** [comment]
Yeah, we may want to make that distinction, same as we do for academic papers : 

Here's a few more too : 

- [Spawned Shelter](https://spawnedshelter.com/) - Erlang and Elixir training with the the best ... 
- [Adopting Erlang](https://adoptingerlang.org/) - Adopting Erlang is an ongoing effort to gather all the resources that will help you use Erlang in a business.
- [Elixir School](https://elixirschool.com/en/) - Elixir School is the premier destination for people looking to learn and master the Elixir programming language.
- [The BEAM book](https://blog.stenmans.org/theBeamBook/) - Documentation and insights on the internals of the Erlang runtime system and the Erlang virtual machine known as the BEAM.
- [BEAM Wisdom](https://beam-wisdoms.clau.se/en/latest/) -  A collection of easy to read (ELI5) articles as well as in-depth knowledge such as VM internals, memory layout, opcodes etc.
- [Erlang Excercisms](https://exercism.io/tracks/erlang) - Learn Erlang through Excercisms
- [Elixir Excercisms](https://exercism.io/tracks/elixir) - Learn Elixir through Excercisms

**Comment by starbelly** [comment]
And some more (courses) : 

[Functional Programming in Erlang,](https://www.futurelearn.com/courses/functional-programming-erlang)  - Learn the theory and practice of functional programming in Erlang, by University of Kent.
[Functional Programming with Erlang](https://www.youtube.com/playlist?list=PLlML6SMLMRgAooeL26mW502jCgWikqx_n) - a free online course for beginners by University of Kent (videos)
[Erlang Master Classes](https://www.youtube.com/playlist?list=PLlML6SMLMRgCaVx42utIleC2aerD504qj) - a free online for advanced Erlang users by University of Kent (videos)
